src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const smallDiscout = 20;
+  const bigDiscount = 50;
+  const rent = 40;
```

Comment:
Be more specific with variable names
```suggestion
  const dailyRent = 40;
```